src/BloomExe/Book/Book.cs

```diff
@@ -1655,6 +1655,17 @@ public void DuplicatePage(IPage page)
 			if (currentPageIndex < 0)
 				return;
 
+			foreach (var span in newpageDiv.SafeSelectNodes(".//span[@class='audio-sentence']").Cast<XmlElement>().ToList())
```

Comment:
If we ever add another class to these spans, like "audio-recorded", this would break and we wouldn't notice. Might be worth changing to "contains()" now.